main.tex

```diff
@@ -108,13 +131,17 @@ \section{Introduction}\label{sec:introduction}
 \begin{enumerate}
  \item Zero determinant strategies are not particularly effective for $N > 2$
  \item Complex strategies can be effective, and in fact can naturally evolve
-     through evolutionary processes to outperform intelligently designed
-     strategies.
- \item Strong resistors specifically evolve or have a handshake mechanism. This
-     has the potential to offer insight in to organisms such as antibacterial
-     resistant bacteria \cite{Davies2010}.
+     through evolutionary processes to outperform designed strategies.
+ \item Strong resistors specifically evolve or have a handshake mechanism.
+ \item Strong invaders are generally cooperative strategies that do not defect
+ first but retaliate to varying degrees of intensity against strategies that defect.
+ \item Strategies evolved to maximize their total payout can be strong invaders
+ and achieve mutual cooperation with many other strategies.
 \end{enumerate}
 
+These finding have the potential to offer insight in to organisms such as antibacterial
```

Comment:
It's a different kind of resistance I think -- resistance to an environmental component rather than invading variants like with a handshake. Handshaking is more like chemical signaling between organisms, tribal fashion, etc.